model/SyncLog/Storage/RdsSyncLogStorage.php

```diff
@@ -0,0 +1,240 @@
+ use common_persistence_SqlPersistence;
+ use Doctrine\DBAL\Connection;
+
+ class RdsSyncLogStorage extends ConfigurableService implements SyncLogStorageInterface
+ {
+     const SERVICE_ID = 'taoSync/RdsSyncLogStorage';
+
+     const OPTION_PERSISTENCE_ID = 'persistenceId';
+
+     const TABLE_NAME = 'synchronisation_log';
+
+     /**
+      * @var string
+      */
+     private $persistenceId = null;
+
+     /**
+      * @var common_persistence_SqlPersistence
+      */
+     private $persistence = null;
+
+     /**
+      * SyncLogService constructor.
+      * @param array $options
+      * @throws MissingOptionException
+      */
+     public function __construct(array $options)
+     {
+         parent::__construct($options);
+
+         if (!$this->hasOption(self::OPTION_PERSISTENCE_ID)) {
+             throw new MissingOptionException('Persistence ID is required option.', self::OPTION_PERSISTENCE_ID);
+         }
+         $this->persistenceId = $this->getOption(self::OPTION_PERSISTENCE_ID);
+     }
+
+     /**
+      * @return common_persistence_SqlPersistence
+      */
+     public function getPersistence()
+     {
+         if ($this->persistence === null) {
+             $this->persistence = $this->getServiceLocator()
+                 ->get(\common_persistence_Manager::SERVICE_ID)
+                 ->getPersistenceById($this->persistenceId);
+         }
+
+         return $this->persistence;
+     }
+
+     /**
+      * @return QueryBuilder
+      */
+     private function getQueryBuilder()
+     {
+         return $this->getPersistence()->getPlatForm()->getQueryBuilder()->from(self::TABLE_NAME);
+     }
+
+     /**
+      * Store SyncLogEntity in rds storage.
+      *
+      * @param SyncLogEntity $entity
+      * @return integer Id of created record.
+      */
+     public function create(SyncLogEntity $entity)
+     {
+
+         $qb = $this->getQueryBuilder();
+         $id = $qb->insert(self::TABLE_NAME)
+             ->values([
+                 SyncLogStorageInterface::COLUMN_SYNC_ID            => $qb->createNamedParameter($entity->getSyncId()),
+                 SyncLogStorageInterface::COLUMN_BOX_ID             => $qb->createNamedParameter($entity->getBoxId()),
+                 SyncLogStorageInterface::COLUMN_ORGANIZATION_ID    => $qb->createNamedParameter($entity->getOrganizationId()),
+                 SyncLogStorageInterface::COLUMN_DATA               => $qb->createNamedParameter(json_encode($entity->getData())),
+                 SyncLogStorageInterface::COLUMN_STATUS             => $qb->createNamedParameter($entity->getStatus()),
+                 SyncLogStorageInterface::COLUMN_REPORT             => $qb->createNamedParameter(json_encode($entity->getReport())),
+                 SyncLogStorageInterface::COLUMN_STARTED_AT         => $qb->createNamedParameter($entity->getStartTime()->format(SyncLogEntity::DATE_TIME_FORMAT))
+             ])
+             ->execute();
+
+         return $id;
```

Comment:
`$qb->insert()->execute()` return number of affected rows but not id of created record.
Also please add a unit test to test that case (return value)